plugins/kv_oauth.ts

```diff
@@ -118,86 +150,37 @@ export function kvOAuthPlugin<
  * // main.ts
  * import { start } from "$fresh/server.ts";
  * import kvOAuthPlugin from "$fresh/plugins/kv_oauth.ts";
- * import { createGitHubOAuth2Client } from "https://deno.land/x/deno_kv_oauth@$VERSION/mod.ts";
+ * import { createGitHubOAuthConfig } from "https://deno.land/x/deno_kv_oauth@$VERSION/mod.ts";
  * import manifest from "./fresh.gen.ts";
  *
  * await start(manifest, {
  *   plugins: [
- *     kvOAuthPlugin(createGitHubOAuth2Client())
+ *     kvOAuthPlugin(createGitHubOAuthConfig())
  *   ]
  * });
  * ```
  */
 export default function kvOAuthPlugin<
```

Comment:
Actually, I reckon we should have two functions: `kvOAuthPlugin()` and `kvOAuthPluginMany()` (or similar). I'm not a fan of multiple argument shapes within a single function. What do we think?